components/eamxx/src/physics/mam/impl/compute_o3_column_density.cpp

```diff
@@ -3,28 +3,27 @@ namespace scream::impl {
 KOKKOS_INLINE_FUNCTION
 void compute_o3_column_density(const ThreadTeam& team, const haero::Atmosphere& atm,
                                const mam4::Prognostics &progs, ColumnView o3_col_dens) {
-  constexpr int nabscol = mam4::gas_chemistry::nabscol;     // number of absorbing densities
   constexpr int gas_pcnst = mam4::gas_chemistry::gas_pcnst; // number of gas phase species
   constexpr int nfs = mam4::gas_chemistry::nfs;             // number of "fixed species"
-  constexpr Real mwdry = 1.0/haero::Constants::molec_weight_dry_air;
+  // constexpr Real mwdry = 1.0/haero::Constants::molec_weight_dry_air;
```

Comment:
All these commented lines are to get rid of compiler warnings due to unused stuff.

Reply:
Also, sometimes I removed the line, sometimes I commented, the rational being that
- if the  variable would be used in a function down below, but said function call is commented since it's not yet ported, then I commented the line declaring the var
- if the variable is _never_ used down below, then I removed the line altogether.